### PR TITLE
Security hub fixes

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do? 

Fixes security hub documentation and feature itself, to allow integration with any deployment options.

Add any further explanations here. 

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing? 

## Testing Checklist

* [ ] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested on AWS with "develop" configuration. 
* [x] If applicable, add screenshots or log transcripts of the feature working
image

### CODE REVIEW

File: monkey/monkey_island/cc/services/reporting/aws_exporter.py [hunk 1]
```diff
@@ -6,13 +6,14 @@
...
 
 from common.cloud.aws.aws_instance import AwsInstance
-from monkey_island.cc.environment import EnvironmentConfig
 from monkey_island.cc.services.reporting.exporter import Exporter
 
 __authors__ = ['maor.rayzin', 'shay.nehmad']
 
 logger = logging.getLogger(__name__)
 
+INFECTION_MONKEY_ARN = "324264561773:product/guardicore/aws-infection-monkey"
```
Comment (acepace) [inline monkey/monkey_island/cc/services/reporting/aws_exporter.py:15]: @itaymmguardicore is there an option that outside people abuse this?

**Comment by acepace** [comment]
This looks good to me but I'm not sure if this can be abused by people

File: monkey/monkey_island/cc/services/reporting/aws_exporter.py (hunk 1 above)
Comment (VakarisZ) [reply monkey/monkey_island/cc/services/reporting/aws_exporter.py:15]: It's just an ID of infection monkey solution in security hub, it's public info
image

**Review by acepace — APPROVED** [review]
(☞ﾟヮﾟ)☞☜(ﾟヮﾟ☜)

**Pull request merged**